_data/projects/owasp-vulnerableapp.yml

```diff
@@ -0,0 +1,18 @@
+name: Owasp VulnerableApp
+desc: As Web Applications are becoming popular these days, there comes a dire need to secure them. Although there are several Vulnerability Scanning Tools, however while developing these tools, developers need to test them. Moreover, they also need to know how well is the Vulnerability Scanning tool performing. As of now, there are little or no such vulnerable applications existing for testing such tools. There are Deliberately Vulnerable Applications existing in the market but they are not written with such an intent and hence lag extensibility, e.g. adding new vulnerabilities is quite difficult. Hence, the developers resort to writing their own vulnerable applications, which usually causes productivity loss and the pain to rework.
```

Comment:
@preetkaran20 I recommend having a short and crisp description. Can we please work on this?